### PR TITLE
Add webp to image-preview

### PR DESCRIPTION
**Pull request by Cr4xy**

### CODE REVIEW

**Comment by acao** [comment]
@Cr4xy if you can sign the CLA with the same email you used to make the commit that would be awesome!!

**Comment by Cr4xy** [comment]
> @Cr4xy if you can sign the CLA with the same email you used to make the commit that would be awesome!!

@acao Done!

File: packages/graphiql-react/src/editor/components/image-preview.tsx [hunk 1]
```diff
@@ -84,5 +84,5 @@ function tokenToURL(token: Token) {
 }
 
 function isImageURL(url: URL) {
-  return /(bmp|gif|jpeg|jpg|png|svg)$/.test(url.pathname);
+  return /(bmp|gif|jpeg|jpg|png|svg|webp)$/.test(url.pathname);
```
Comment (dimaMachina) [inline packages/graphiql-react/src/editor/components/image-preview.tsx:87]:
```suggestion
  return /\.(bmp|gif|jpeg|jpg|png|svg|webp)$/.test(url.pathname);
```

File: .changeset/pink-singers-glow.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+---
+"@graphiql/react": patch
+"graphiql": patch
```
Comment (dimaMachina) [inline .changeset/pink-singers-glow.md:3]:
```suggestion
"@graphiql/react": minor
"graphiql": minor
```

File: packages/graphiql-react/src/editor/components/image-preview.tsx [hunk 3]
```diff
@@ -84,5 +84,5 @@ function tokenToURL(token: Token) {
 }
 
 function isImageURL(url: URL) {
-  return /(bmp|gif|jpeg|jpg|png|svg)$/.test(url.pathname);
+  return /\.(bmp|gif|jpeg|jpg|png|svg|webp)$/.test(url.pathname);
```
Comment (dimaMachina) [inline packages/graphiql-react/src/editor/components/image-preview.tsx:87]:
```suggestion
  return /\.(bmp|gif|jpe?g|png|svg|webp)$/.test(url.pathname);
```

**Review by dimaMachina — APPROVED** [review]
lgtm

**Pull request merged**